assets/scripts/src/choices.js

```diff
@@ -485,98 +486,103 @@ class Choices {
    * @private
    */
   render() {
-    this.currentState = this.store.getState();
-
-    // Only render if our state has actually changed
-    if (this.currentState !== this.prevState) {
-      // Choices
-      if (
-        this.currentState.choices !== this.prevState.choices ||
-        this.currentState.groups !== this.prevState.groups ||
-        this.currentState.items !== this.prevState.items
-      ) {
-        if (this.isSelectElement) {
-          // Get active groups/choices
-          const activeGroups = this.store.getGroupsFilteredByActive();
-          const activeChoices = this.store.getChoicesFilteredByActive();
-
-          let choiceListFragment = document.createDocumentFragment();
-
-          // Clear choices
-          this.choiceList.innerHTML = '';
-
-          // Scroll back to top of choices list
-          if (this.config.resetScrollPosition) {
-            this.choiceList.scrollTop = 0;
-          }
+    if (!this._inMultipleStoreChanges){
```

Comment:
ESLint would have caught this space-before-blocks issue. Not major, but this would show as a delta between the fork's version in the dist folder, and what we're using in RWC.

Reply:
Yes, agreed.